mlrun/runtimes/databricks_job/databricks_runtime.py

```diff
@@ -197,14 +196,13 @@ def _pre_run(self, runspec: RunObject, execution):
             if value:
                 task_parameters[key] = value  # in order to handle reruns.
         runspec.spec.parameters["task_parameters"] = task_parameters
-        current_file = os.path.abspath(__file__)
-        current_dir = os.path.dirname(current_file)
-        databricks_runtime_wrap_path = os.path.join(
-            current_dir, "databricks_wrapper.py"
-        )
-        with open(databricks_runtime_wrap_path, "r") as databricks_runtime_wrap_file:
-            wrap_code = databricks_runtime_wrap_file.read()
-            wrap_code = b64encode(wrap_code.encode("utf-8")).decode("utf-8")
+        code = """\n
+from mlrun.runtimes.databricks_job import databricks_wrapper
+
+def run_mlrun_databricks_job(context,task_parameters: dict, **kwargs):
+        databricks_wrapper.run_mlrun_databricks_job(context, task_parameters, **kwargs)
+"""
+        wrap_code = b64encode(code.encode("utf-8")).decode("utf-8")
```

Comment:
you can set `code = b""" ... """` and avoid using `code.encode(..)` as it is initialized as bytearray upon declaration